api.yaml

```diff
@@ -102,8 +102,19 @@ tags:
       These endpoints need to respond quickly as our cache building processes will hit them in large bursts when we are rebuilding the entire cache.
       Note that our cache is only used for customer experience reasons. Before placing orders we will always validate that the request dates and/or timeslots still offer availability.
 
-      Availability must be provided consistently for at least 2 months in advance, up to 6 months.
-
+      The availability endpoints should behave as follows:
+          - At least 2 months should be provided. In case the request for availability is for more days in advance than are actually able to be provided, the API should not fail but just send the amount of days available.
+          - The request will be for the next 6 natural months from the day of the request (not more, not less in the current version). This means that depending on the month it could be a different total amount of days.
+
+      Example request:
+          - Parameters: 
+            - start_date: 20/01/2020
+            - end_date: 20/07/2020
+            - API only has 2 months of availability
+          Expected output:
+            - Availability for those 2 months that can be provided, never an error.
```

Comment:
```suggestion
```
      The availability endpoints should behave as follows:
      
        - At least 2 months should be provided. In case the request for availability is for more days in advance than are actually able to be provided, the API should not fail but just send the number of days available.
        - The request will be for the next 6 natural months from the day of the request (not more, not less in the current version). This means that depending on the month it could be a different total amount of days.

      Example request:
    
        - Parameters: 
          - start_date: 20/01/2020
          - end_date: 20/07/2020
          - API only has 2 months of availability
        - Expected output:
          - Availability for those 2 months that can be provided, never an error.
```
Markdown fix.

Reply:
Supplier can still return an error if the requested range is too wide.
```
2009 | Incorrect date | Eg. Cannot use the past date or Given date is too far ahead in the future (max 6 months)
```